runtime/jcl/common/system.c

```diff
@@ -181,14 +181,10 @@ Java_java_lang_System_getSysPropBeforePropertiesInitialized(JNIEnv *env, jclass
 		sysPropValue = getDefinedArgumentFromJavaVMInitArgs(vmInitArgs, "com.ibm.autocvt");
 		if (NULL == sysPropValue) {
 			/* As part of better handling of JEP400 constraints on z/OS, the com.ibm.autocvt property
-			 * determines whether we convert input file I/O based on file tagging. If not explicitly specified,
-			 * the property defaults to true, unless file.encoding is set to COMPAT. */
+			 * determines whether file I/O considers file tagging. If not explicitly specified,
+			 * the property defaults to true, unless file.encoding is set. */
```

Comment:
Sorry, I missed it earlier: the end of a multi-line comment should be on a line by itself.
```c
			/* As part of better handling of JEP400 constraints on z/OS, the com.ibm.autocvt property
			 * determines whether file I/O considers file tagging. If not explicitly specified,
			 * the property defaults to true, unless file.encoding is set.
			 */
```

Reply:
Fixed under [5dbfd87](https://github.com/eclipse-openj9/openj9/pull/19924/commits/5dbfd8760028fcfe60506530cacb9589a4e439a1).